webcompat/form.py

```diff
@@ -428,15 +444,20 @@ def build_formdata(form_object):
     if extra_labels:
         metadata_keys.append('extra_labels')
 
+    clean = re.compile('<.*?>')
+    problem_type_stripped = re.sub(clean, '', get_radio_button_label(
+        form_object.get('problem_category'), problem_choices))
+    browser_tested_on = form_object.get('browser_test')
```

Comment:
There is [no new additional tests](https://github.com/webcompat/webcompat.com/blob/f3cce74a422c51d492ef9a1772108001ad5aa105/tests/unit/test_form.py#L178-L196) testing the changed logic of `build_form_data`